libs/core/langchain_core/output_parsers/transform.py

```diff
@@ -37,9 +38,13 @@ async def _atransform(
     ) -> AsyncIterator[T]:
         async for chunk in input:
             if isinstance(chunk, BaseMessage):
-                yield self.parse_result([ChatGeneration(message=chunk)])
+                yield await run_in_executor(
```

Comment:
Should we create a single thread with an event loop and delegate to it instead of re-recreating on every chunk?

Reply:
Hmm no the event loop manages its own thread pool, no need for us to repeat that work